### PR TITLE
fix(products): update products added first in Open Prices

### PR DESCRIPTION
**Pull request by dq18**
### What
Products first entered in Open Prices were not updated once added in OFF. Their "source" is set to None/null.

### Potential issue
- if user enters a product with barcode that is not a food product but has the same barcode as a food product (is that possible?)

### Fixes bug(s)
- #243

### CODE REVIEW

File: app/tasks.py [hunk 1]
```diff
@@ -146,7 +151,12 @@ def import_product_db(db: Session, batch_size: int = 1000) -> None:
                         Product.updated == None,  # noqa: E711, E501
                     )
                 )
-                .values(**item)
+                .values(
+                    source=Flavor.off
```
Comment (raphodn) [inline app/tasks.py:155]: `source=item.get("source", Flavor.off)`

File: app/tasks.py [hunk 2]
```diff
@@ -137,7 +137,12 @@ def import_product_db(db: Session, batch_size: int = 1000) -> None:
             execute_result = db.execute(
                 update(Product)
                 .where(Product.code == product_code)
-                .where(Product.source == Flavor.off)
+                .where(
+                    or_(
+                        Product.source == Flavor.off,
```
Comment (raphodn) [inline app/tasks.py:142]: maybe add a comment to explain why we have these 2 usecases ?

File: app/tasks.py (hunk 1 above)
Comment (raphodn) [reply app/tasks.py:155]: actually the `item.get("source")` will always be None, as "source" is not in OFF_FIELDS

Comment (raphodn) [reply app/tasks.py:155]: so i would add this above in the "item" creation section
`item["source"] = Flavor.off`

Comment (dq18) [reply app/tasks.py:155]: Ideed. I wanted a conditional in the values on `Product.source is None` but that did not work out so I thought `item` would do. Thought about using a case statement but not sure how that works in SQLAlchemy in the update method.

File: app/tasks.py (hunk 2 above)
Comment (dq18) [reply app/tasks.py:145]: Added

**Pull request merged**